self_hosting_machinery/webgui/selfhost_model_resolve.py

```diff
@@ -21,6 +21,27 @@ def completion_resolve_model(inference_queue: InferenceQueue) -> Tuple[str, str]
     return completion_model, ""
 
 
+def dynamic_resolve_model(model_name: str, inference_queue: InferenceQueue) -> Tuple[str, str]:
```

Comment:
hmm...model resolving not really needed here, you should check that model is in inference_queue.models_available() and if it's not log and raise exception